modules/jwt_verify_lib/0.0.0-202230517-b59e807/patches/jwt_verify_lib.patch

```diff
@@ -0,0 +1,194 @@
+diff --git a/BUILD b/BUILD
+index 63c69bc..0a936b4 100644
+--- a/BUILD
++++ b/BUILD
+@@ -23,11 +23,11 @@ cc_library(
+         "jwt_verify_lib/verify.h",
+     ],
+     deps = [
+-        "//external:abseil_flat_hash_set",
```

Comment:
Could you try to submit this patch upstream first? We don't have to wait for long before merging this as is, not at least we can link the PR.